src/resources/views/crud/fields/inc/translatable_icon.blade.php

```diff
@@ -1,3 +1,19 @@
-@if (method_exists($crud->model, 'isTranslatableAttribute') && ($crud->model->isTranslatableAttribute($field['name']) || (isset($field['store_in']) ? $crud->model->isTranslatableAttribute($field['store_in']) : false)) && config('backpack.crud.show_translatable_field_icon'))
-<i class="la la-flag-checkered pull-{{ config('backpack.crud.translatable_field_icon_position') }}" style="margin-top: 3px;" title="This field is translatable."></i>
-@endif
\ No newline at end of file
+@php
+    // if field name is array we check if any of the arrayed fields is translatable
+    $translatable = false;
+    if($crud->model->translationEnabled()) {
+        foreach((array) $field['name'] as $field_name){
+            if($crud->model->isTranslatableAttribute($field_name)) {
+                $translatable = true;
+            }
+        }
+
+        if(isset($field['store_in']) && $crud->model->isTranslatableAttribute($field['store_in'])) {
+                $translatable = true;
+        }
+    }
```

Comment:
Now that I look to this once again, I'm thinking about bit wise operations 😅

```suggestion
    $translatable = false;
    if($crud->model->translationEnabled()) {
        foreach((array) $field['name'] as $field_name){
            $translatable |= $crud->model->isTranslatableAttribute($field_name));
        }

        $translatable |= isset($field['store_in']) && $crud->model->isTranslatableAttribute($field['store_in']));
    }
```
For those not aware of it, **it is less readable**, but it's a lot more efficient, because when `$translatable` becomes `true` it will ignore next calls, it keeps the loop, but it will not go on crud, model, and check if field is translatable ever again.

It's the equivalent to:
```
if(!$translatable && isset($field['store_in']) && $crud->model->isTranslatableAttribute($field['store_in'])) {
    $translatable = true;
}
```

But much more succinct. What do you guys think? I'm not fully certain of using this 😬

Reply:
Pfff... I like that it's shorter. But like you said... it's a lot less readable. Let's stay out of bitwise until it becomes "the norm".